speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/mainscreens/DataSelectionMenu.cpp

```diff
@@ -167,6 +175,8 @@ void* DataSelectionMenuInit(void* p_nextMenu)
     m_dataToStoreControl[2] = GfuiMenuCreateCheckboxControl(s_scrHandle, param, PRM_HUMAN_DATA, nullptr, ChangeHumanStorage);
     m_dataToStoreControl[3] = GfuiMenuCreateCheckboxControl(s_scrHandle, param, PRM_INTRV_DATA, nullptr, ChangeInterventionStorage);
     m_dataToStoreControl[4] = GfuiMenuCreateCheckboxControl(s_scrHandle, param, PRM_META_DATA, nullptr, ChangeMetaDataStorage);
+    GfuiMenuCreateButtonControl(s_scrHandle, param, "database", nullptr, DatabaseSettingsMenuActivate);
```

Comment:
Order matters. Either place the button below the checkboxes in the xml or make the buttonControl of the button before the controls for the checkboxes. This way the arrowkey input controls scroll nicely through the options